.ado/pipelines/templates/jobs-workload-deploy.yaml

```diff
@@ -27,9 +27,6 @@ jobs: # using jobs so they each run in parallel
           $frontDoorId = $globalInfraDeployOutput.frontdoor_id_header.value
           echo "*** Retrieved Azure Front Door Header ID $frontDoorId"
 
-          $frontDoorFqdn = $globalInfraDeployOutput.frontdoor_fqdn.value
```

Comment:
This shouldn't exist anymore. It should have been dropped in PR #307.

Reply:
good spot. maybe was some merge conflict. remove it